src/Controller/AdminRegistrationController.php

```diff
@@ -32,6 +32,9 @@ class AdminRegistrationController extends LeftAndMain
 
     private static $url_segment = 'mfa';
 
+    /**
+     * @deprecated 5.4.0 Will be removed without equivalent functionality to replace it
+     */
     private static $ignore_menuitem = true;
```

Comment:
This class changed to not be a subclass of `LeftAndMain` and therefore doesn't need this at all anymore.